package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "frosting",
+  "version": "1.0.0",
+  "description": "Every cake needs a little frosting! Add some flair to your project\r by adding some Frosting style.",
+  "main": "dist/index.css",
```

Comment:
this file isn't included in the git repo, rather, you should do an `npm run build`, and then do an `npm publish`, to create and release this file.